imagelab/DisplayImage.java

```diff
@@ -68,7 +74,7 @@ public DisplayImage(ImgProvider imp, String title, boolean slow) {
         int pixwidth  = imp.pixwidth;
         img = getToolkit().createImage(
                 new MemoryImageSource(pixwidth, pixheight, showpix, 0, pixwidth));
-        pane        = new DisPanel(img);
+        pane        = new DynaPanel(img);
```

Comment:
Initialization of new DynaPanel object using img object. The purpose here is to use the functionality of DynaPanel that allows for dropdown menus to be selected while music is playing and image is synchronizing.